### PR TITLE
Add `marketplace[interregional_retailer]` to captures and capture refunds

### PR DESCRIPTION
**Pull request by cbobach**
We are extending our API to support Visa's Foreign Retail Indicator (FRI) for marketplaces. We have chosen to rephrase this to `marketplace[interregional_retailer]`.

The parameter will be required when a merchant is a marketplace. The conditional requirement will be the same for both Visa and Mastercard.

### CODE REVIEW

**Review by mt-clearhaus — APPROVED** [review]
Nice :+1:

File: website/content/gateway/api_reference/changes/changes.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### Add marketplace regional purchase indicator to captures and capture refunds
+The `marketplace` concept has been added to captures and capture refunds. The
+`marketplace[regional_purchase]` parameter is optional with default `true`.
```
Comment (mt-clearhaus) [inline website/content/gateway/api_reference/changes/changes.md:15]:
```suggestion
`marketplace[regional_purchase]` parameter is required if the merchant is a
marketplace.
```

Because we are considering this option.

File: website/content/gateway/api_reference/resources/capture_refunds/capture_refunds.md [hunk 2]
```diff
@@ -27,6 +27,13 @@ These are refunds of one or more captures made on an authorization and shall not
 {{% regex_optional %}}Optional{{% /regex_optional %}}
 {{% /description_details %}}
 
+{{% description_term %}}marketplace[regional_purchase]{{% regex %}}(true|false){{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Indicates whether a marketplace purchase is regional or not.
+
+Default: `true`.
+{{% regex_optional %}}Optional{{% /regex_optional %}}
```
Comment (mt-clearhaus) [inline website/content/gateway/api_reference/resources/capture_refunds/capture_refunds.md:34]:
```suggestion
{{% regex_optional %}}Required if the merchant is a marketplace.{{% /regex_optional %}}
```

File: website/content/gateway/api_reference/resources/captures/captures.md [hunk 3]
```diff
@@ -28,6 +28,13 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
 {{% regex_optional %}}Optional{{% /regex_optional %}}
 {{% /description_details %}}
 
+{{% description_term %}}marketplace[regional_purchase]{{% regex %}}(true|false){{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Indicates whether a marketplace purchase is regional or not.
+
+Default: `true`.
+{{% regex_optional %}}Optional{{% /regex_optional %}}
```
Comment (mt-clearhaus) [inline website/content/gateway/api_reference/resources/captures/captures.md:35]:
```suggestion
{{% regex_optional %}}Required if the merchant is a marketplace.{{% /regex_optional %}}
```

File: website/content/gateway/api_reference/changes/changes.md (hunk 1 above)
Comment (mt-clearhaus) [reply website/content/gateway/api_reference/changes/changes.md:15]: We should also add a timestamp for when the requirement is enforced.

Comment (cbobach) [reply website/content/gateway/api_reference/changes/changes.md:15]: Should we run this by the @clearhaus/partner team to land on a date for enforcement?

**Comment by cbobach** [comment]
Rendered after latest changes:
image

File: website/content/gateway/api_reference/changes/changes.md (hunk 1 above)
Comment (mt-clearhaus) [reply website/content/gateway/api_reference/changes/changes.md]: Yeah, let's do that :+1:

Comment (mt-clearhaus) [reply website/content/gateway/api_reference/changes/changes.md]: I've asked internally and am awaiting answers :heavy_check_mark:

Comment (mt-clearhaus) [reply website/content/gateway/api_reference/changes/changes.md]: For people following this: The requirement will be enforced for new marketplaces right away (when deployed).

File: website/content/gateway/api_reference/changes/changes.md [hunk 4]
```diff
@@ -10,6 +10,11 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### Add marketplace regional purchase indicator to captures and capture refunds
+The `marketplace` concept has been added to captures and capture refunds. The
+`marketplace[regional_purchase]` parameter is required if the merchant is a
+marketplace.
```
Comment (mt-clearhaus) [inline website/content/gateway/api_reference/changes/changes.md:16]: Naming is hard. We are currently considering replacing `regional_purchase` with `interregional_retailer`.

```suggestion
#### Add marketplace interregional retailer indicator to captures and capture refunds
A `marketplace` concept has been added to captures and capture refunds. The
`marketplace[interregional_retailer]` parameter is required if the merchant is
a marketplace.
```

File: website/content/gateway/api_reference/resources/capture_refunds/capture_refunds.md [hunk 5]
```diff
@@ -27,6 +27,12 @@ These are refunds of one or more captures made on an authorization and shall not
 {{% regex_optional %}}Optional{{% /regex_optional %}}
 {{% /description_details %}}
 
+{{% description_term %}}marketplace[regional_purchase]{{% regex %}}(true|false){{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Indicates whether a marketplace purchase is regional or not.
```
Comment (mt-clearhaus) [inline website/content/gateway/api_reference/resources/capture_refunds/capture_refunds.md:31]:
```suggestion
{{% description_term %}}marketplace[interregional_retailer]{{% regex %}}(true|false){{% /regex %}}{{% /description_term %}}
{{% description_details %}}Indicates whether or not a marketplace retailer outside EEA, Gibraltar, and the UK is involved in the transaction.
```

File: website/content/gateway/api_reference/resources/captures/captures.md [hunk 6]
```diff
@@ -28,6 +28,12 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
 {{% regex_optional %}}Optional{{% /regex_optional %}}
 {{% /description_details %}}
 
+{{% description_term %}}marketplace[regional_purchase]{{% regex %}}(true|false){{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Indicates whether a marketplace purchase is regional or not.
```
Comment (mt-clearhaus) [inline website/content/gateway/api_reference/resources/captures/captures.md:32]:
```suggestion
{{% description_term %}}marketplace[interregional_retailer]{{% regex %}}(true|false){{% /regex %}}{{% /description_term %}}
{{% description_details %}}Indicates whether or not a marketplace retailer outside EEA, Gibraltar, and the UK is involved in the transaction.
```

**Review by mt-clearhaus — COMMENTED** [review]
Possible replacement of `regional_purchase` with `interregional_retailer`.
